Remove explicit dependency on riak_pb

engelsanchez opened the pull request:
riak_search depends on riak_api, which has a rebar dependency on
riak_pb, so this one is not needed. Moreover, it's conflicting with the
one in riak_api, causing compilation problems in riak.

engelsanchez committed:
Remove explicit dependency on riak_pb

riak_search depends on riak_api, which has a rebar dependency on
riak_pb, so this one is not needed. Moreover, it's conflicting with the
one in riak_api, causing compilation problems in riak.

seancribbs commented:
I would actually remove riak_api altogether. It is a transitive dep through riak_kv.

engelsanchez committed:
Remove redundant riak_api dependency

riak_api is already a dependency of riak_kv, which is already listed in
rebar.config

engelsanchez commented:
By George, you are right! Obviously this rebar configuration has received little love. Removed that and tested from a fresh compile.

seancribbs commented:
:+1:
